File: DayTwo/Java/QuestionFour.java
public class QuestionFour{

public static void printNumbersReverseVersion(){
        String reverse = "";

        for(int count = 0; count <= 100; count++){

            if(count % 3 == 0)
                   System.out.println("hello");
            if(count % 5 == 0)
                   System.out.println("world");
            if(count % 7 == 0)
                   System.out.println("hello");
            
            else{
                String newCount = Integer.toString(count);
                int length = newCount.length();
                for(int index = 0; index < newCount.length(); index++){
                    reverse = newCount.charAt(index) + reverse;
                }
                  System.out.println(reverse);
            }
                     
            
        }
}

public static void main(String...ugo){
    printNumbersReverseVersion();
}

File: DayTwo/JavaScript/QuestionFour.js

let printNumbersReverseVersion = ()=>{
        reverse = "";

        for(let count = 0; count <= 100; count++){

            if(count % 3 == 0)
                   console.log("hello");
            if(count % 5 == 0)
                   console.log("world");
            if(count % 7 == 0)
                   console.log("hello");
            
            else{
                let newCount = String(count);
                let length = newCount.length
                for(let index = 0; index < newCount.length; index++){
                    reverse = newCount.charAt(index) + reverse;
                }
                  console.log(reverse);
            }
                     
            
        }
}


    printNumbersReverseVersion();


